packages/utilities/src/utilities/parseDsn.ts

```diff
@@ -56,7 +57,17 @@ export const parseDsn = (dsn: string): ConnectionOptions => {
         .describe(
           'Specifies the location for the secret key used for the client certificate.',
         ),
-      sslmode: z.enum(['disable', 'no-verify', 'require']).optional(),
+      sslmode: z
+        .enum([
+          'allow',
```

Comment:
`allow` is not listed in [`pg` documentation](https://github.com/brianc/node-postgres/blob/master/packages/pg-connection-string/README.md#tcp-connections) so I don't think we should support it.

Also, it is not used in the switch line 147.